assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BEZIER } from '../../../../animation';
+import {
+  themeHelpers,
+  Icons,
+  NotificationBubble,
+} from '../../../../design-system';
+
+const Button = styled.button`
+  ${themeHelpers.reset.button}
+  ${themeHelpers.centerContent}
+  ${themeHelpers.focusableOutlineCSS}
+  ${themeHelpers.expandTextPreset(({ label }, sizes) => label[sizes.MEDIUM])}
+  ${({ theme }) => css`
+    color: ${theme.colors.fg.primary};
+    border-radius: ${theme.borders.radius.small};
+    border: 1px solid ${theme.colors.bg.tertiary};
+  `}
+  padding: 1px 14px 1px 4.5px;
+  transform-origin: 50% 50%;
+
+  &:active {
+    transform: scale(0.99);
```

Comment:
the secondary button type as it exists in figma today has a purple active color, that's really the only override you'd need here (unless I missed something). so add `background-color: ${({ theme }) => theme.colors.bg.secondary};`